core/editor/data/globals/endpoints/typography.php

```diff
@@ -13,13 +14,20 @@ protected function get_kit_items() {
 
 		$kit = Plugin::$instance->kits_manager->get_active_kit_for_frontend();
 
-		$system_items = $kit->get_settings_for_display( 'system_typography' );
-		$custom_items = $kit->get_settings_for_display( 'custom_typography' );
+		// Use raw settings that doesn't have default values.
+		$meta_key = PageManager::META_KEY;
+		$kit_raw_settings = $kit->get_meta( $meta_key );
```

Comment:
Instead of accessing the meta directly, you can use:

```suggestion
		$kit_raw_settings = $kit->get_data( 'settings' );
```